README.md

```diff
@@ -98,26 +99,32 @@ The event name and what's in the `data` for each event handler:
   * `event` The raw event
   * `senderId` The ID of the sender
   * `session` [A Session object](#the-session-object) you can mutate
-  * `text` Message content, `event.message.text` for text events
+  * `payload` Message content, `event.message.text` for text events
```

Comment:
The `text` event can still do `.on('text', {text, payload})` without affecting `.on('message.text', {text})`. It does create an inconsistency problem though where `text` means original text one place and original text everywhere else.

There was a NAMINGTHINGS  comment I can't find anymore about `text`, `originalText`(`payload`) and `normalizedText`. The current pr is written where:
* `text` = normalized text
* `payload` = original text

So we either need to decide to break backwards compatibility or changes the names so
* `text` = original text
* `normalizedText`? = normalized text

Making `text` ambiguous isn't a good option